src/core/validations/message.rs

```diff
@@ -424,6 +461,12 @@ pub fn validate_user_data_add_body(body: &UserDataBody) -> Result<(), Validation
         UserDataType::Github => {
             validate_github_username(&body.value)?;
         }
+        UserDataType::UserDataPrimaryAddressEthereum => {
+            validate_user_data_primary_address_ethereum(&body.value)?;
+        }
+        UserDataType::UserDataPrimaryAddressSolana => {
+            validate_user_data_primary_address_solana(&body.value)?;
```

Comment:
There's a `validate_eth_address` and `validate_sol_address` being used by the verification validation. We should use the same functions. Inconsistencies would causes issues where a verification that was accepted wouldn't allowed to be used as the primary address

Reply:
changed!